HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
 from P2 import *
+# from P2a import make_mandelbrot_image_and_hist
 
-# Your code here
+import findspark
+findspark.init()
+import numpy as np
+import pyspark
+
+sc = pyspark.SparkContext(appName='mandelbrot')
+
+
+x_values = sc.parallelize(np.random.permutation(2000), 10)
+y_values = sc.parallelize(np.random.permutation(2000), 10)
+coords = x_values.cartesian(y_values)
+
+def make_mandelbrot_image_and_hist(coords_rdd, hist_fname):
```

Comment:
Not that it really matters very much, but what was the point in creating this function in P2.py and then copy-pasting it here? I see you have the import of this function commented out in line 2, I'm just wondering why.